indra/llui/lllayoutstack.cpp

```diff
@@ -409,12 +408,15 @@ void LLLayoutStack::updateLayout()
 	llassert(total_visible_fraction < 1.05f);
 
 	// don't need spacing after last panel
-	space_to_distribute += panelp ? ll_round((F32)mPanelSpacing * panelp->getVisibleAmount()) : 0;
+	{
+		LLLayoutPanel* panelp = mPanels.empty() ? nullptr : mPanels.back();
+		space_to_distribute += panelp ? ll_round((F32)mPanelSpacing * panelp->getVisibleAmount()) : 0;
+	}
```

Comment:
Simpler to express this as:

    if (! mPanels.empty())
    {
        space_to_distribute += ll_round(F32(mPanelSpacing) * mPanels.back()->getVisibleAmount());
    }